src/layout/Footer.js

```diff
@@ -3,8 +3,7 @@ import PropTypes from 'prop-types'
 
 const Footer = props =>
   <div style={{marginBottom: `2rem`, clear: `both`, width: `100%`}}>
-    <a href={props.outProxy + props.moreInformationUrl}> See more expression data at Expression Atlas.</a>
-    <br/>This expression view is provided by <a href={props.outProxy + props.atlasUrl}>Expression Atlas</a>.
+    This page is a summary of the data held in <a href={props.outProxy + props.atlasUrl}>Expression Atlas</a> for this gene; click <a href={props.outProxy + props.moreInformationUrl}>here </a>for the full record.
```

Comment:
The second link has a space that should the link tag. Also, be aware that very long lines have a negative impact on readability. We use 120 character-wide lines in most of our code. That’s a sensible value and you can configure your IDE or editor to display a right margin to keep that under control.